lib/logger.js

```diff
@@ -104,14 +102,17 @@ function createLogger (options) {
   return { logger, hasLogger: true }
 }
 
-function implementsLog4jInterface (logger) {
-  const methods = ['info', 'error', 'debug', 'fatal', 'warn', 'trace']
-  for (let i = 0; i < methods.length; i += 1) {
-    if (!logger[methods[i]] || typeof logger[methods[i]] !== 'function') {
-      return false
-    }
+function isValidLogger (logger) {
```

Comment:
The original implementation should have had a jsdoc block attached. That's my miss. As you're working on this, it is a good opportunity to fix that. Please add a jsdoc block to document this function's purpose, parameters, returns, and throws.